ui/src/app/nav.tsx

```diff
@@ -43,13 +60,16 @@ export function Aside({ isOpen, setIsOpen }: { isOpen: boolean, setIsOpen: Dispa
                             <label htmlFor="textarea">
                                 Enter or upload CSR in PEM format below
                             </label>
-                            <textarea id="csr-textarea" name="textarea" rows={10} placeholder="-----BEGIN CERTIFICATE REQUEST-----" onChange={handleTextChange} value={CSRPEMString}/>
+                            <textarea id="csr-textarea" name="textarea" rows={10} placeholder="-----BEGIN CERTIFICATE REQUEST-----" onChange={handleTextChange} value={CSRPEMString} />
                         </div>
                         <div className="p-form__group row">
                             <input type="file" name="upload" accept=".pem" onChange={handleFileChange}></input>
```

Comment:
CSR's often have `.csr` as the file suffix, we should accept those

Reply:
done